src/OrchardCore.Modules/OrchardCore.Menu/MenuShapes.cs

```diff
@@ -6,10 +6,12 @@
 using OrchardCore.DisplayManagement.Descriptors;
 using OrchardCore.DisplayManagement.Shapes;
 using OrchardCore.Menu.Models;
+using OrchardCore.Modules;
 using OrchardCore.Mvc.Utilities;
 
 namespace OrchardCore.Menu
 {
+    [Feature("OrchardCore.Menu")]
```

Comment:
Need to be tried but I think the feature attribute is not needed when related to the module main feature.

Reply:
Done